platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gossip/chatter/ChatterGossip.java

```diff
@@ -103,36 +103,37 @@ public class ChatterGossip extends AbstractGossip {
     private final List<StoppableThread> chatterThreads = new LinkedList<>();
     private final ChatterEventMapper chatterEventMapper = new ChatterEventMapper();
     private final SequenceCycle<EventIntakeTask> intakeCycle;
-    private final Clearable clearAllPipelines;
+    private final Clearable clearAllInternalPipelines;
```

Comment:
the `clearAllPipelines` member variables in the other sync class should also be renamed to `clearAllInternalPipelines`

Reply:
Updated for all implementations